lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -205,7 +205,7 @@ class EntityGenerator
         Types::BLOB               => 'string',
         Types::DECIMAL            => 'string',
         Types::GUID               => 'string',
-        Type::JSON_ARRAY          => 'array',
+        Types::JSON               => 'array',
```

Comment:
```suggestion
        'json_array'              => 'array',
        Types::JSON               => 'array',
```

This class is flagged for removal anyway. I would not introduce breaking changes at this point. Let's simply inline the deprecated constant so we get a clean build on DBAL 3.